src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -244,6 +246,29 @@ export const CreateTransaction = ({
     setIsMemoVisible(!!found);
   };
 
+  const checkIfAccountIsFunded = async () => {
+    const { toAccountId } = formData;
+
+    if (!toAccountId || !StrKey.isValidEd25519PublicKey(toAccountId)) {
+      onInput({
+        ...formData,
+        isAccountFunded: true,
```

Comment:
I like how we're resetting it if someone removes the account id or puts a wrong one on accident